lib/elixir_google_scraper/scraper.ex

```diff
@@ -9,6 +9,16 @@ defmodule ElixirGoogleScraper.Scraper do
   alias ElixirGoogleScraper.Repo
   alias ElixirGoogleScraper.Scraper.{CSVKeyword, Keyword}
 
+  def paginated_keywords(user, params \\ %{}) do
+    user
+    |> user_keywords
+    |> Repo.paginate(params)
+  end
+
+  defp user_keywords(user) do
```

Comment:
Let's move it to bottom of the file 👍  and probably rename to `user_keywords_query` 🤔  because it is to build the query